index.html

```diff
@@ -889,6 +884,76 @@ <h2 class='informative'>Vary</h2>
       </section>
     </section>
 
+    <section id='external-content'>
+      <h3>External LDP-NR Content</h3>
+      <p class='informative'>
+        Variability among client types and locations may mean that LDP-NR
+        content is addressed in ways that are external to LDP-S but not resolvable
+        by all clients. This specification describes the use of <code>message/external-body</code>
+        Content-Type value to signal, on POST or PUT, that the LDP-S should should not consider the
+        request entity to be the LDP-NR's content, but that a Content-Type value will signal a
+        name or address at which the content might be retreived. The <code>url</code>
+        and <code>local-file</code> type values motivate this specification, but a Fedora LDP-S
+        may support any type parameters per the requirements for advertisement and
+        rejection specified here.
+      </p>
+      <section id='message-external-body-accept-post'>
+        <p>
+          LDP-S that support LDP-NR with <code>message/external-body</code>
+          MUST advertise that support in the <code>Accept-Post</code> response
+          header for each supported <code>type</code> parameter of supported Content-Types.
+        </p>
+      </section>
+      <section id='message-external-body-unsupported-type'>
+        <p>
+          LDP-S receiving requests that would create or update a LDP-NR
+          with a <code>message/external-body</code> with an unsupported
+          <code>type</code> parameter MUST respond with HTTP 415 UNSUPPORTED MEDIA TYPE.
+          In the case that a Fedora LDP-S does not support external LDP-NR content, all
+          <code>message/external-body</code> messages must be rejected with HTTP 415.
+        </p>
+      </section>
+      <section id='message-external-body-response-headers'>
+        <p>
+          LDP-S receiving requests that would create or update a LDP-NR
+          with a <code>message/external-body</code> MUST NOT accept the
+          request if it cannot guarantee all of the response headers required
+          by the LDP-NR interaction model in this specification.
+        </p>
+      </section>
+      <section id='external-content-content-location'>
+        <p>
+          LDP-NR GET and HEAD responses SHOULD include a Content-Location: header with a URI
+          representation of the location of the external content if the LDP-S is proxying
+          the content.
+        </p>
+      </section>
+      <section id='external-content-expires'>
+        <p>
+          Per [[!RFC2017]], all <code>message/external-body</code> Content-Type values
```

Comment:
I am seeing no mention of `expires` or `expiration` in RFC2017... but do in [RFC1521](https://tools.ietf.org/html/rfc1521)